sql/moz-fx-data-shared-prod/search_terms_derived/search_term_data_validation_reports_v1/init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS
+  `mozdata.search_terms_unsanitized_analysis.prototype_data_validation_reports_v1`(
```

Comment:
```suggestion
  `moz-fx-data-shared-prod.search_terms_derived.prototype_data_validation_reports_v1`(
```

Reply:
Close! Pulling "prototype" out of the name. Change pushed!